Code/Tools/SceneAPI/SceneData/ManifestBase/SceneNodeSelectionList.cpp

```diff
@@ -104,6 +104,50 @@ namespace AZ
                 }
             }
 
+            bool SceneNodeSelectionListVersionConverter(
+                AZ::SerializeContext& serializeContext, AZ::SerializeContext::DataElementNode& classElement)
+            {
+                // Already at latest version, nothing to upgrade.
+                if (classElement.GetVersion() >= 3)
+                {
+                    return true;
+                }
```

Comment:
This is not needed, it would need to be removed if this class updates to version 4.

The version converter will never get called if it matches the current version.

Reply:
Guess I should wrap the whole conversion block in `if (classElement.GetVersion() < 3)`? That's what I originally had, and flipped it to this so that I didn't have so much indented code, but it's a fair point that with the way the code is written now, it would be harder to add more version conversions at the end.